packages/kbn-failed-test-reporter-cli/failed_tests_reporter/get_failures.ts

```diff
@@ -105,3 +109,14 @@ function getCommandLineFromReport(report: TestReport) {
     return report.testsuite?.$['command-line'] || '';
   }
 }
+
+function getOwner(report: TestReport): CodeOwnership {
+  // TODO-TRE: Remove comments and update logic
+  // if ('testsuites' in report) {
+  //   return report.testsuites?.testsuite?.[0]?.$['command-line'] || '';
+  // } else {
+  //   return report.testsuite?.$['command-line'] || '';
+  // }
+  if ('testsuites' in report) return report.testsuites?.testsuite?.[0]?.testcase?.[0].$.owners;
```

Comment:
Is it safe to say every existing `testcase` should have non-empty array and first item to have `owners` property?
Personally I would expected to have owner as `undefined` or `unknown` rather than not mentioned at all

Reply:
Not sure yet! lol. I'm placing this back in draft as I noticed a problem